code/modules/clothing/clothing_vr.dm

```diff
@@ -181,6 +181,13 @@
 		standing.layer = BODY_LAYER + 15 // 15 is above tail layer, so will not be covered by taurbody.
 	return standing
 
+/obj/item/clothing/suit/apply_accessories(var/image/standing)
+	if(LAZYLEN(accessories))
+		for(var/obj/item/clothing/accessory/A in accessories)
+			var/mutable_appearance/M = new(A.get_mob_overlay())
+			M.pixel_x = 16 //Opposite of the pixel_x on the suit (-16) from taurization to cancel it out and puts the accessory in the correct place on the body.
+			standing.add_overlay(M)
```

Comment:
I don't see any conditions to make this only shift by 16 on taurs only?

Passing an MA into add_overlay seems to do *weird things* and can cause *weird issues*. Namely SSoverlays skipping the rest of the queued overlays that tick. It's not noticeable in local testing where having a significant queue is basically impossible, but I did a similar thing on the live server and people started getting random things stuck to their bodies. Do you remember that shift? ;p

I'd suggest using an image instead! You can do the exact same code here and just replace the MA with an image instead, as they also have pixel stuff (MAs are actually a child of /image but don't tell anyone). You can just do var/image/I = A.get_mob_overlay(), I.pixel_x = 16, standing.add_overlay(I) (get_mob_overlay returns an image so you don't need to like, turn it into another image)

Reply:
Fuck I thought I had the taurized condition in there, that would've been disastrous. Good call regarding the SSoverlays queue, always gotta account for the voodoo

Testing and committing in a sec